### PR TITLE
Fix Linting Errors

### LINKED ISSUE #504: Broken Linting in file apps/viewer/init.js
**Describe the bug**
Broken Linting in the file apps/viewer/init.js

**To Reproduce**
1. run npm run lint && npm run lint-fix
2. See error

**Expected behavior**
There shouldn't be any linting errors

**Screenshots**

**Desktop (please complete the following information):**
 - OS: linux

**Smartphone (please complete the following information):**

**Additional context**
Working on a fix

### PR DESCRIPTION
**Pull request by AkhileshAdithya**
Fixed errors that aren't automatically fixed with npm run lint-fix

## Description
- Changed for each loop to a for loop to avoid lint error (must be wrapped in an if clause)
- Broke down long lines to abide by lint max length condition

## Motivation and Context
**Why is this change required? What problem does it solve?
- fixes #504 
- fixes lint errors

## How Has This Been Tested?
**Please describe in detail how you tested your changes.
- ran npm run lint && npm run lint-fix

## Types of changes
**What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
**Go over all the following points, and put an `x` in all the boxes that apply.
**(If you're unsure about any of these, don't hesitate to ask. We're here to help!)
- [x] My code follows the code style of this project. [not exactly sure though]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/viewer/init.js [hunk 1]
```diff
@@ -248,19 +248,23 @@ function initCore() {
...
           const items = $UI.layersViewer.setting.categoricalData.human.items;
           var dataType = null;
-          for(const key in items){
+          for (const [key, value] of Object.entries(animals)) {
+            // split condition to condition 1 and condition 2 to abide by eslint(max-len)
             dataType = key;
-            if (items.hasOwnProperty(key)&&Array.isArray(items[key].items)&&items[key].items.some(i=>i.item.id == $UI.annotPopup.data.id)) break;
+            var condition1 = items.hasOwnProperty(key)&&Array.isArray(items[key].items);
+            var condition2 = items[key].items.some((i)=>i.item.id == $UI.annotPopup.data.id);
+            if (condition1&&condition2) break;
```
Comment (birm) [inline apps/viewer/init.js:256]:
```suggestion
            var isRelevant = items.hasOwnProperty(key)&&Array.isArray(items[key].items) &&
                             items[key].items.some((i)=>i.item.id == $UI.annotPopup.data.id);
            if (isRelevant) break;
```

File: apps/viewer/init.js [hunk 2]
```diff
@@ -248,19 +248,23 @@ function initCore() {
...
-        $UI.annotPopup.dataType = null;
-        $UI.annotPopup.dataType = data.provenance && data.provenance.analysis && data.provenance.analysis.source && data.provenance.analysis.source=='human'?
-        getCateName($UI.annotPopup.data.id):null;
 
+        $UI.annotPopup.dataType = null;
+        var condition1 = data.provenance && data.provenance.analysis && data.provenance.analysis.source;
+        var condition2 = data.provenance.analysis.source=='human';
+        if ( condition1 && condition2 ) {
+          $UI.annotPopup.dataType = getCateName($UI.annotPopup.data.id);
+        }
```
Comment (birm) [inline apps/viewer/init.js:267]:
```suggestion
        var isHuman = data.provenance && data.provenance.analysis && data.provenance.analysis.source;
                         data.provenance.analysis.source=='human';
        if ( isHuman ) {
          $UI.annotPopup.dataType = getCateName($UI.annotPopup.data.id);
        }
```

**Review by birm — COMMENTED** [review]
I would prefer avoiding non-descriptive names as possible

**Comment by birm** [comment]
Is this replicating https://github.com/camicroscope/caMicroscope/commit/004a58fd52435769df42f1eabc105071068d59d2 ?

**Comment by AkhileshAdithya** [comment]
> Is this replicating [004a58f](https://github.com/camicroscope/caMicroscope/commit/004a58fd52435769df42f1eabc105071068d59d2) ?

Ahh, yes, My bad... I was working on the main branch.

**Comment by birm** [comment]
No problem, thank you for taking the time to try to help code quality, even if it didn't get merged this time :)

**Pull request closed without merging**